File: 백준 Java/Step1/src/Pro1_11.java
import java.util.Scanner;

public class Pro1_11 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		int hunb = b/100;
		int tenb = b/10 - hunb*10;
		int oneb = b - tenb*10 - hunb*100;
		System.out.println(a*oneb);
		System.out.println(a*tenb);
		System.out.println(a*hunb);
		System.out.println(a*b);
	}

}

File: 백준 Python/Step1/pro1_11.py
a = int(input())
b = int(input())
hunb = int(b/100)
tenb = int(b/10)-hunb*10
oneb = b - tenb*10 - hunb*100
print(a*oneb, a*tenb, a*hunb, a*b, sep='\n')
